docs/contributing/contributor-guidelines.md

```diff
@@ -1,3 +1,31 @@
+
+Nearly all work occurs in public repos and is open to new contributors. A full list of repos can be found in the Decred [GitHub org](https://github.com/decred). A list of projects under active development and descriptions of them can be found on the [Projects](../../projects) page. 
+
+If new to the project, be sure to check out the user-facing [docs](https://docs.decred.org) to get an understanding of the basics of Decred. Then find a project that interests you and that you feel you have the skills to contribute to. Once you've found a project you're interested in, find an issue you'd like to work on. GitHub Issues tagged with `good first issue` are a good place to start. 
+
+## Contributing
+
+Once you've found an issue you'd like to work on, announce your plans **before** starting work. This avoids duplicate work, minimizes the chances you're working on something that might not fit with the consensus of the community, and permits discussions that can help you achieve your goal. Most projects have a dedicated chat channel for discussing project-specific work. 
+
+Some repos will also have their own specific guidelines. For instance, the dcrd repo has fairly extensive [Code Contribution Guidelines](https://github.com/decred/dcrd/blob/master/docs/code_contribution_guidelines.md).
+
+## GitHub
+
+Whie different repos may have slightly different GitHub processes, below are the basic steps common to all repos.
+
+1. Fork the repo on GitHub.
+1. Clone the forked repo. 
+1. Make your changes in a local branch.
+1. When your changes are ready to be reviewed (or you just want input from other devs), push your branch to your forked repo and open a Pull Request (PR) on GitHub against the main repo. If your changes address an Issue, put `Closes #` or `Fixes #`in the PR description. This will allow GitHub to automatically close the Issue when the PR is merged. 
+1. All changes must be reviewed and receive at least one approval before they can be merged. Only team members can give official approval, but comments from other users are encouraged.
+1. If a reviewer requests changes, make those changes in the local branch you were working in. Then push those changes to your forked repo. This will automatically update the PR in a way that allows the reviewer to see the changes you've made.
+1. Once approved, your PR can be merged into master. To keep history clean, only non-fast-forward merges are allowed (i.e. we want a linear history). If a change consists of multiple commits, those commits typically must be squashed into a single commit. If there is a reason to include multiple commits in a PR, that will be considered on a case-by-case basis. 
+1. If your PR consists of a single commit (or multiple commits that can be automatically squased by GitHub), and is caught up with master, the reviewer will merge your PR. If your branch is too far behind master, you may be asked to rebase your commit and push again. The reviewer will then merge your PR. 
```

Comment:
squashed*

Reply:
"is caught up with master" => "has no conflicts with master"

Reply:
I think the final sentence "The reviewer will then merge your PR." is unnecessary/repetitive. Maybe phrase differently or remove?

Reply:
Accepted all of these fixes/suggestions